frontend/src/pages/projects/screens/spawner/SpawnerPage.tsx

```diff
@@ -90,7 +92,13 @@ const SpawnerPage: React.FC<SpawnerPageProps> = ({ existingNotebook }) => {
     defaultStorageClassName,
   );
 
-  const [envVariables, setEnvVariables] = useNotebookEnvVariables(existingNotebook);
+  const [envVariables, setEnvVariables, envVariableloaded] =
+    useNotebookEnvVariables(existingNotebook);
+  const { deletedConfigMap, deletedSecret } = deletedConfigMapOrSecretVariables(
+    existingNotebook,
+    envVariables,
+  );
+
```

Comment:
in my opinion, becuase the deletedConfigMap and deletedSecret are only used in the alrt component, why not call this function in that component and then pass in existingNotebook and  envVariables instead. keeps things cleaner on the spawner page